src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
You use the expression: `(days * 40`) in other places, so let's create a separate variable for this.